.github/issue_template.md

```diff
@@ -0,0 +1,19 @@
+---
+name: openHABian issue template
+about: A template issue for the openHABian repository.
+labels: new
+---
+## Issue information:
+Please be as detailed as possible to help us help you. Also, please be sure to
+check that your issue has not already been reported to the [openHAB community](https://community.openhab.org/t/13379)
```

Comment:
cannot "report" on the forum so reword that
also point to https://community.openhab.org/t/how-to-ask-a-good-question-help-us-help-you/58396

Reply:
Better?

Reply:
yes